docs/style-guide.md

```diff
@@ -323,8 +323,8 @@ impl ListFormat {
 }
 ```
 
-This rule may become a tradeoff in rare cases of APIs which optionally allocates. For example an API that in 95% of cases only needs a reference, but in the remaining 5% it needs to allocate, should be explicitly marked as such.
-An example of such API in the `stdlib` is `HashSet::get_or_insert` and `HashSet::get_or_insert_owned`:
+This rule may become a tradeoff in rare cases of APIs which optionally allocate. For example, if an API only needs a reference in 95% of cases, but in the remaining 5% needs to allocate, it should be explicitly marked as such.
```

Comment:
Shouldn't the try_new function in this example take the Locale by value anyway so it doesn't have to close and so the caller is allowed to move if it can move instead of clone?

Reply:
Which example do you refer to? The `ListFormat` example above is recommending exactly that marking it as `GOOD`, and discouraging the reference with `BAD`.

Reply:
I see.  I think I'm used to BAD being before GOOD.  Consider this comment an optional suggestion to switch the order of the good and bad examples.